README.md

```diff
@@ -174,3 +175,23 @@ release:
   variables:
     INPUT_VERSION: yarn changeset version
 ```
+
+You may also want to run `yarn install` after the `changeset verion` command to update the yarn.lock in the version MR. You need to disable immutable lock file setting using an env variable:
```

Comment:
Personally, I don't think we should mention the following example, because it is not much related to this module IMO, I've never run `yarn install` after `changeset version` before without any issue.

Reply:
I don't have any issue with removing it but the pipelines for the people that use yarn v2 and pnp will fail if they don't do this.  
Actually my build and publish pipeline (`yarn install` at the beginning of the pipeline) failed every time I merged the Version Packages MR because the `yarn.lock` was out of sync with the actual package versions.

Reply:
Can you paste some error messages here?